catalyst-gateway/tests/api_tests/integration/test_assets.py

```diff
@@ -1,27 +1,36 @@
 import json
 import os
+import codecs
 
 import pytest
 from loguru import logger
 from api.v1 import cardano
-from functools import reduce
 
 
 @pytest.mark.preprod_indexing
 def test_persistent_ada_amount_endpoint():
     # could the file from https://github.com/input-output-hk/catalyst-storage/blob/main/cardano-asset-preprod.json
     ASSETS_DATA_PATH = os.environ["ASSETS_DATA_PATH"]
+    # 5% failure rate
+    ALLOWED_FAILURE_RATE = 0.05
```

Comment:
Given failures are 0.45 make this 0.1
```suggestion
    ALLOWED_FAILURE_RATE = 0.1
```